templates/action.epp

```diff
@@ -2,10 +2,11 @@
 $action_name,
 $type,
 $facility,
+$dont_linebreak_actions,
 $config
 | -%>
 # <%= $action_name %>
-<% if $facility != "default" and size($config) < 3 { -%>
+<% if $facility != "default" and size($config) < 3 or $dont_linebreak_actions { -%>
   <%= sprintf( '%-30s %s%s%s', $facility, 'action(type="',$type,'" ' )-%>
```

Comment:
Is this a complicated version of:
```suggestion
  <%= sprintf('%-30s action(type="%s" ', $facility, $type) -%>
```

:thinking:

Reply:
I'm afraid that I'm don't know what line 10 does so I can't comment on your suggestion.

Reply:
Sure, I was trying to think about what could be done to make this more simple to read, and spotted this. I guess some refactoring could make the code more simple…  Not necessary something that must be done in this PR, so I will mark this as resolved.